### PR TITLE
bsp: imx-common: standalone_build_kernel: Fix indentation

### PR DESCRIPTION
**Pull request by s-hemer**
The code block wasn't correctly formatted. Fixes 9badff3380f40

### CODE REVIEW

**Comment by s-hemer** [comment]
BTW this code block seems to be somehow out of context: bullet point before is "building kernel", following is about where to find the results. Imho installing kernel modules is an more advanced task and should go to the end of the list (behind re-building DTs)?!

**Comment by tremmet** [comment]
> BTW this code block seems to be somehow out of context: bullet point before is "building kernel", following is about where to find the results. Imho installing kernel modules is an more advanced task and should go to the end of the list (behind re-building DTs)?! Potentially, this is also a duplicate info as it is part of the following section, so completely remove here?

Feel free to create a pull request or write a ticket in our jira project.

**Comment by tremmet** [comment]
@s-hemer don't forget to add reviewers to your pull request :)

**Review by tremmet — APPROVED** [review]

File: source/bsp/imx-common/development/standalone_build_kernel.rsti [hunk 1]
```diff
@@ -39,7 +39,7 @@ Build Kernel
 *  Install kernel modules to e.g. NFS directory:
 
   .. code-block:: console
```
Comment (BHahn42618) [inline source/bsp/imx-common/development/standalone_build_kernel.rsti:41]: This line is also missing one space.
```suggestion
   .. code-block:: console
```

**Review by BHahn42618 — CHANGES_REQUESTED** [review]

File: source/bsp/imx-common/development/standalone_build_kernel.rsti (hunk 1 above)
Comment (s-hemer) [reply source/bsp/imx-common/development/standalone_build_kernel.rsti]: Fixed

**Review by BHahn42618 — APPROVED** [review]

**Review by tremmet — APPROVED** [review]

**Pull request merged**